src/presentation/http/router/note.ts

```diff
@@ -205,6 +225,13 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
 
     const addedNote = await noteService.addNote(content as JSON, userId as number, parentId); // "authRequired" policy ensures that userId is not null
```

Comment:
Maybe use JSDoc for this comment?

Reply:
this is not part of my pr , so it was approved i guess